Libs/MRML/Logic/vtkMRMLSliceLogic.cxx

```diff
@@ -549,11 +549,23 @@ void vtkMRMLSliceLogic::ProcessMRMLLogicsEvents()
     plane->SetOrigin(outPt3);
 
```

Comment:
This could be cleaner using std::max. Additionally, the unused min and max macros at the top of this file could be removed.